internal/common/run_yarn.js

```diff
@@ -9,11 +9,11 @@ function runYarn(cwd, command = "") {
   });
 }
 
-function yarnShellCommand(cwd, command = "") {
+function yarnShellCommand(cwd, command = "", yarn = "yarn") {
```

Comment:
How about not having a default value? (you'd have to update usage in a couple of places to explicitly call it with "install" as the command).
```suggestion
function yarnShellCommand(cwd, command, yarn) {
```